desktop.css

```
@@ -0,0 +1,62 @@
+@media screen and (min-width: 720px) {
+    .pharmacist {
+        display: block
+    }
+    .info {
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+    }
+    .info > :not(:first-child) {
+        margin-top: 0;
+    }
+    body {
+        color: black;
+        background-color: white;
+        text-align: center;
+        font-family: "Merriweather", serif;
+    }
+    img {
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+        width: 65%;
+    }
+    .pharmacist {
+        display: none;
+    }
+    .tagline {
+        color: #1A237E;
+        font-weight: bold;
+        font-size: 24px;
+    }
+    .services {
+        vertical-align: center;
+        font-size: 20px;
+        margin-top: 8px;
+    }
+    .info {
+        background-color:#7F0000;
+        color: white;
+        font-family: "Merriweather", serif;
+        padding: 40px;
+        font-weight: normal;
+        font-size: 16px;
+        margin-top: 40px;
+    }
+    .info > :not(:first-child) {
+     margin-top: 32px;
+    }
+    .contact > :not(:last-child),
+    .hours > :not(:last-child),
+    .location > :not(:last-child) {
+        margin-bottom: 14px;
+    }
+    .header {
+        font-size: 24px;
+        font-weight: bold;
+    }
+    .label {
+        font-size: 16px;
+        font-weight: bold;
+    }
+}
```

"New line at the end of the file" thing again
```suggestion
}

```